client/src/pages/Profile/ProfilePhoto/ProfilePhoto.tsx

```diff
@@ -0,0 +1,61 @@
+import Grid from '@material-ui/core/Grid';
+import { Avatar, Button, Typography } from '@material-ui/core';
+import useStyles from './useStyles';
+import { useState } from 'react';
+import DeleteIcon from '@material-ui/icons/Delete';
+import { useSnackBar } from '../../../context/useSnackbarContext';
+
+export default function ProfilePhoto(): JSX.Element {
```

Comment:
I think we should utilise a library like React Dropzone for drag and drop file upload, could you do that?